src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -102,46 +181,18 @@ const ChartsHolder = ({ projections, region, chartId }: ChartsHolderProps) => {
     scrollToRecommendations();
   }, [chartId, metricRefs, isRecommendationsShareUrl]);
 
-  const stats = projection ? projections.getMetricValues() : {};
   const initialFipsList = useMemo(() => {
-    return [projections.primary.fips];
-  }, [projections.primary.fips]);
-
-  const recommendationsIntro = getDynamicIntroCopy(
-    projection,
-    projections.locationName,
-    projections.getMetricValues(),
-  );
-
-  const recommendationsMainContent = getRecommendations(
-    projection,
-    mainContent.recommendations,
-  );
-
-  const recommendsShareUrl = getRecommendationsShareUrl(region);
-
-  const alarmLevel = projections.getAlarmLevel();
-  const recommendsShareQuote = getShareQuote(
-    projections.locationName,
-    alarmLevel,
-  );
+    return projections?.primary?.fips ? [projections.primary.fips] : [];
+  }, [projections]);
 
-  const recommendationsFeedbackForm = `https://can386399.typeform.com/to/WSPYSGPe#source=can&id=${uuidv4()}&fips=${
-    projection.fips
-  }`;
+  const ccviScores = useCcviForFips(region.fipsCode);
 
-  // TODO(Chelsi): make these 2 functions less redundant?
-  const recommendationsFedModalCopy = getModalCopyWithFedLevel(
-    projection,
-    projections.locationName,
-    projections.getMetricValues(),
-  );
+  if (!locationSummary) {
```

Comment:
finally after all the hooks, if there's no location summary we can't render at all.

Reply:
yeah, hooks cannot be called conditionally, I think we should move the hooks inside the components they belong to, but we can take care of that during the redesign (some of them are needed by more than one block I think)